components/airbrake/sources/new-error-occurred/new-error-occurred.mjs

```diff
@@ -0,0 +1,78 @@
+import airbrake from "../../airbrake.app.mjs";
+
+export default {
+  name: "New Error Occurred",
+  version: "0.0.1",
+  key: "airbrake-new-error-occurred",
+  description: "Emit new event for each error occurred.",
+  type: "source",
+  dedupe: "unique",
+  props: {
+    airbrake,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      static: {
+        intervalSeconds: 15 * 60, // 15 minutes
+      },
+    },
+    projectId: {
+      propDefinition: [
+        airbrake,
+        "projectId",
+      ],
+    },
+  },
+  methods: {
+    emitEvent(data) {
+      this._setLastNoticeId(data.id);
+
+      this.$emit(data, {
+        id: data.id,
+        summary: `New payment with id ${data.id}`,
+        ts: Date.parse(data.created_at),
+      });
+    },
+    _setLastNoticeId(id) {
+      this.db.set("lastNoticeId", id);
+    },
+    _getLastNoticeId() {
+      return this.db.get("lastNoticeId");
+    },
+  },
+  hooks: {
+    async deploy() {
+      const { notices } = await this.airbrake.getProjectNotices({
+        projectId: this.projectId,
+        params: {
+          page: 1,
+        },
+      });
+
+      notices.slice(10).reverse()
```

Comment:
You can add the limit in the request instead of here. I think it might be a better performance 
![image](https://user-images.githubusercontent.com/5638699/200859817-153d92c8-b439-45ca-98c2-352a3b986e39.png)

Reply:
On deploy hook we just emit 10 events for default, this is the reason of the `slice()`!

Reply:
but you can limit to 10 in the request instead of the slice(), can't you?